cloudformation/cw-events-substack.cfn.json

```diff
@@ -0,0 +1,97 @@
+{
+  "Outputs": {
+    "HealthSQSQueueARN": {
+      "Description": "ARN of SQS health queue",
+      "Value": {
+        "Fn::GetAtt": [
+          "HealthEventQueue",
+          "Arn"
+        ]
+      }
+    },
+    "HealthSQSQueueName": {
+      "Description": "Name of SQS health queue",
+      "Value": {
+        "Fn::GetAtt": [
+          "HealthEventQueue",
+          "QueueName"
+        ]
+      }
+    }
+  },
+  "Resources": {
+    "EventsSQSPolicy": {
+      "Properties": {
+        "PolicyDocument": {
+          "Id": "EventsQueuePolicy",
+          "Statement": [
+            {
+              "Action": [
+                "sqs:SendMessage"
+              ],
+              "Effect": "Allow",
+              "Principal": {
+                "Service": [
+                  "events.amazonaws.com"
+                ]
+              },
+              "Resource": "*",
```

Comment:
I thought it would be good to leave this permission open so that users can add new rules that push to health queue easily, based on some not intuitive behaviors of `PolicyDocument` like [here](https://stackoverflow.com/questions/35443237/aws-cloudformation-2-sqspolicy-for-a-single-queue).

However, I agree that we should probably scope down to specific resource for added security